src/qt/veil.cpp

```diff
@@ -590,13 +590,12 @@ int main(int argc, char *argv[])
     Q_INIT_RESOURCE(veil);
     Q_INIT_RESOURCE(veil_locale);
 
-    BitcoinApplication app(*node, argc, argv);
 #if QT_VERSION > 0x050100
     // Generate high-dpi pixmaps
     QApplication::setAttribute(Qt::AA_UseHighDpiPixmaps);
 #endif
 #if QT_VERSION >= 0x050600
```

Comment:
Not enough to hold it up, but I would prefer if we start using the syntax `QT_VERSION_CHECK(5,6,0)`, especially considering QT_VERSION is actually built from a QT_VERSION_CHECK call.